packages/effects-core/src/plugins/text/text-item.ts

```
@@ -211,49 +217,140 @@ export class TextComponent extends MaskableGraphic implements ITextComponent {
+            x += letterSpace;
+          }
+          x += measureChar(str);
           this.maxLineWidth = Math.max(this.maxLineWidth, x);
         }
-      } else {
-        if (((x + textMetrics) > width && i > 0) || str === '\n') {
-          lineCount++;
-          this.maxLineWidth = Math.max(this.maxLineWidth, x);
-          x = 0;
-          charCountInLine = 0; // 重置行字符计数
+      }
```

ESLint

[error] Expected blank line before this statement.

(padding-line-between-statements)